### PR TITLE
python3Packages.cairocffi: 1.1.0 -> 1.2.0

### PR DESCRIPTION
**Pull request by teto**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115305` at 35453bdf run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/1739ba9ce5f29bfe6b1f041b9b14bcf3">mirage-im</a></li></li>
    <li><a href="https://gist.github.com/ab895e6bb9ae32b855fcc386cb66421c">python38Packages.cairosvg</a></li></li>
    <li><a href="https://gist.github.com/65eb1fa5b42546d61fd78c2ef8341515">python38Packages.pygal</a></li></li>
    <li><a href="https://gist.github.com/e10fa915434aa52be98d518e04180a51">python38Packages.weasyprint</a></li></li>
    <li><a href="https://gist.github.com/535ebcda85675ec9f5f96eac3268c29f">xml2rfc</a></li> (python38Packages.xml2rfc)</li>
    <li><a href="https://gist.github.com/007f2ea09d78e9d22bf838536ea3c330">python39Packages.cairosvg</a></li></li>
    <li><a href="https://gist.github.com/806594777656e68016f60c2761476d13">python39Packages.pygal</a></li></li>
    <li><a href="https://gist.github.com/8640a72a1e99b9ec52ca60a723f729af">sourcehut.metasrht</a></li></li>
  </ul>
</details>
<details>
  <summary>9 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/f8ef886812dc68d67bd74bed46f7a264">python38Packages.cairocffi</a></li></li>
    <li><a href="https://gist.github.com/916b73682e6e293659a2c7c3c3c92bc4">python38Packages.graphite-web</a></li></li>
    <li><a href="https://gist.github.com/632a4ca5ba1905654ac6fa9276cba7cf">python38Packages.graphite_api</a></li></li>
    <li><a href="https://gist.github.com/b0ea60aebfc9828c0f6ca7687ef6157b">python38Packages.influxgraph</a></li></li>
    <li><a href="https://gist.github.com/6c7f0f658193478ac82abb5f3bc6a2ea">python39Packages.cairocffi</a></li></li>
    <li><a href="https://gist.github.com/ed9513f42df32a57f7bc50dc69c32502">python39Packages.graphite-web</a></li></li>
    <li><a href="https://gist.github.com/745d472dbe7ede95d9d8008eff0b5862">python39Packages.graphite_api</a></li></li>
    <li><a href="https://gist.github.com/037592652fd6b9d36249a3a78cfdffc2">python39Packages.influxgraph</a></li></li>
    <li><a href="https://gist.github.com/7a1d22b6e2fb638dbbb15eec0398afc0">qtile</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/cairocffi/generic.nix:48:5:
```
   |
48 |     maintainers = with maintainers; [];
   |     ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
> Result of `nixpkgs-review pr 115305` at [35453bd](https://github.com/NixOS/nixpkgs/commit/35453bdfa6af4cde3a31ce3c8aa95ff794d5ed98) run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 
> 8 packages failed to build:
> 9 packages built:
> 1 suggestion:

Fixed in https://github.com/NixOS/nixpkgs/pull/115307

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115305` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
17 packages built:
</summary>
<ul>
<li>mirage-im</li>
<li>python38Packages.cairocffi</li>
<li>python38Packages.cairosvg</li>
<li>python38Packages.graphite-web</li>
<li>python38Packages.graphite_api</li>
<li>python38Packages.influxgraph</li>
<li>python38Packages.pygal</li>
<li>python38Packages.weasyprint</li>
<li>xml2rfc (python38Packages.xml2rfc)</li>
<li>python39Packages.cairocffi</li>
<li>python39Packages.cairosvg</li>
<li>python39Packages.graphite-web</li>
<li>python39Packages.graphite_api</li>
<li>python39Packages.influxgraph</li>
<li>python39Packages.pygal</li>
<li>qtile</li>
<li>sourcehut.metasrht</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
mirage-im:
<details>


warning: unclear-gpl
`lgpl3` is a deprecated license, check if project uses `lgpl3Plus` or `lgpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/networking/instant-messengers/mirage/default.nix:53:5:

```
   |
53 |     license = licenses.lgpl3;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>
</details>

File: pkgs/development/python-modules/cairocffi/default.nix [hunk 1]
```diff
@@ -12,15 +12,16 @@
 , glibcLocales
 , cairo
 , cffi
+, numpy
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cairocffi/default.nix:15]: Does this break 0.9? Can that version maybe be removed?

**Comment by teto** [comment]
```
18 packages updated:
metasrht mirage python38Packages.cairocffi (1.1.0 → 1.2.0) python3.8-CairoSVG python3.8-graphite-api python3.8-graphite-web python3.8-influxgraph python3.8-pygal python3.8-weasyprint python3.8-xml2rfc python39Packages.cairocffi (1.1.0 → 1.2.0) python3.9-CairoSVG python3.9-graphite-api python3.9-graphite-web python3.9-influxgraph python3.9-pygal qtile xml2rfc

$ nix --experimental-features nix-command build --no-link --keep-going --option build-use-sandbox relaxed -f /home/teto/.cache/nixpkgs-review/rev-d1dc02322ae15ec34be233d1f84d600c9c939101/build.nix
17 packages built:
mirage-im python38Packages.cairocffi python38Packages.cairosvg python38Packages.graphite-web python38Packages.graphite_api python38Packages.influxgraph python38Packages.pygal python38Packages.weasyprint xml2rfc python39Packages.cairocffi python39Packages.cairosvg python39Packages.graphite-web python39Packages.graphite_api python39Packages.influxgraph python39Packages.pygal qtile sourcehut.metasrht
```

**Comment by SuperSandro2000** [comment]
```
nix-build -A python27Packages.cairocffi
error: undefined variable 'numpy' at /home/hotpi/.cache/nixpkgs-review/pr-115305-1/nixpkgs/pkgs/development/python-modules/cairocffi/generic.nix:26:19
(use '--show-trace' to show detailed location information)
```

**Comment by risicle** [comment]
`nixpkgs-review` happy macos 10.14

**Comment by SuperSandro2000** [comment]
python38Packages.xml2rfc evntually core dumps:

```
       > ............................................................/nix/store/zqycq4ajl75ardhhiixndn9wkprxbhl6-stdenv-linux/setup: line 1310:   308 Trace/breakpoint trap
(core dumped) pytest -k 'not test_font_stretch'
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115305` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python38Packages.weasyprint: log was empty</li>
<li>xml2rfc: log was empty</li>
<li>sourcehut.metasrht: log was empty</li>
  </ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>(python38Packages.xml2rfc): log was empty</li>
  </ul>
</details>
<details>
<summary>
14 packages built:
</summary>
<ul>
<li>mirage-im</li>
<li>python38Packages.cairocffi</li>
<li>python38Packages.cairosvg</li>
<li>python38Packages.graphite-web</li>
<li>python38Packages.graphite_api</li>
<li>python38Packages.influxgraph</li>
<li>python38Packages.pygal</li>
<li>python39Packages.cairocffi</li>
<li>python39Packages.cairosvg</li>
<li>python39Packages.graphite-web</li>
<li>python39Packages.graphite_api</li>
<li>python39Packages.influxgraph</li>
<li>python39Packages.pygal</li>
<li>qtile</li>
</ul>
</details>

**Pull request merged**